site2/docs/admin-api-partitioned-topics.md

```diff
@@ -80,6 +88,12 @@ command and specifying the topic name as an argument. This is an example command
 $ bin/pulsar-admin topics create persistent://my-tenant/my-namespace/my-topic
 ``` 
 
+> #### Note
+>
+> It's only allow to create non partitioned topic of name contains suffix '-partition-' followed by numeric value like
+> 'xyz-topic-partition-10', if there's already a partitioned topic with same name, in this case 'xyz-topic', and has
+> number of partition larger then that numeric value in this case 10. Else creation of such topic will fail.
```

Comment:
partition index is start from `0`, so `xyz-topic-partition-10` indicate that the number of partition is no less than 11.

Reply:
updated.